### PR TITLE
Feature/bump sdk version 4.0.0

### PR DESCRIPTION
**Pull request by sergiomarqmoura**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -11,12 +12,45 @@ final ERROR_MESSAGE = "You have not defined an app key for the Onfido api."
...
+        release {
+            keyAlias 'release'
+            keyPassword 'release'
+            storeFile file('/Users/sergiomoura/development/release.jks')
+            storePassword 'release'
+        }
+    }
     compileSdkVersion 27
-
+    buildToolsVersion '27.0.3'
```
Comment (dlwcz) [inline app/build.gradle]: I think it's not required anymore, is it?

File: app/build.gradle [hunk 2]
```diff
@@ -11,12 +12,45 @@ final ERROR_MESSAGE = "You have not defined an app key for the Onfido api."
...
+    runOnEachAssemble = true
+    maxMethodCount = 120000
+}
 
 android {
+    signingConfigs {
+        release {
+            keyAlias 'release'
+            keyPassword 'release'
+            storeFile file('/Users/sergiomoura/development/release.jks')
```
Comment (dlwcz) [inline app/build.gradle]: the path is not generic I guess :D

File: app/build.gradle (hunk 1 above)
Comment (sergiomarqmoura) [reply app/build.gradle]: you're right, removed

File: app/build.gradle (hunk 2 above)
Comment (sergiomarqmoura) [reply app/build.gradle]: What a shame :( removed

**Review by dlwcz — APPROVED** [review]

File: app/build.gradle [hunk 3]
```diff
@@ -11,12 +12,36 @@ final ERROR_MESSAGE = "You have not defined an app key for the Onfido api."
...
+    repositories {
+        jcenter()
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.getkeepsafe.dexcount:dexcount-gradle-plugin:0.6.4'
+    }
+}
+
+dexcount {
```
Comment (rfreitas) [inline app/build.gradle]: not sure if it's useful for a sample app? We probably need another test app which uses gradle external import. but for integrator is better to have a simple sample app to use as reference.

**Review by rfreitas — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 3 above)
Comment (sergiomarqmoura) [reply app/build.gradle]: I see. Removed this.
Regarding another app for DEX counting, let's discuss that

**Review by rfreitas — APPROVED** [review]
LGTM!

**Pull request merged**